### PR TITLE
updated for OCPQE-18363(remove hypershift-hosted tag)  for verification-tests

### PR DESCRIPTION
**Pull request by ShudiLi**
remove hypershift-hosted tag from cucushift cases which using client pod inside the cluster

@lihongan @melvinjoseph86 please help review the changes, thanks.

### CODE REVIEW

**Comment by ShudiLi** [comment]
to address [OCPQE-18363](https://issues.redhat.com/browse/OCPQE-18363)

File: features/routing/route.feature [hunk 1]
```diff
@@ -526,7 +521,7 @@ Feature: Testing route
       | http://<%= route("reen", service("service-secure")).dns(by: user) %>/ |
       | -k |
     Then the step should succeed
-    And the output should contain:
+   And the output should contain:
```
Comment (liangxia) [inline features/routing/route.feature:524]: Looks like this is an unexpected change ?

**Comment by lihongan** [comment]
/lgtm
Thank you!

**Pull request merged**